app/src/main/java/com/sunasterisk/rememory/ui/addwork/AddWorkFragment.kt

```diff
@@ -0,0 +1,62 @@
+package com.sunasterisk.rememory.ui.addwork
+
+import android.annotation.SuppressLint
+import android.widget.PopupMenu
+import com.sunasterisk.rememory.BaseFragment
+import com.sunasterisk.rememory.R
+import kotlinx.android.synthetic.main.fragment_add_work.*
+
+class AddWorkFragment : BaseFragment(), AddWorkContract.View {
+    override val layoutResource get() = R.layout.fragment_add_work
+    override fun initComponent() {
+        showClassifyMenu()
+        showEvaluateMenu()
+    }
+
+    @SuppressLint("SetTextI18n")
+    private fun showClassifyMenu() {
+        buttonClassify.setOnClickListener {
+            val classifyMenu = PopupMenu(activity, buttonClassify)
+            classifyMenu.apply {
+                menuInflater.inflate(R.menu.classify_menu, classifyMenu.menu)
+                setOnMenuItemClickListener { item ->
+                    when (item.itemId) {
+                        R.id.classifyWork -> buttonClassify.text =
+                            getString(R.string.title_classify_work)
+                        R.id.classifyRelax -> buttonClassify.text =
+                            getString(R.string.title_classify_relax)
+                        R.id.classifyFamily -> buttonClassify.text =
+                            getString(R.string.title_classify_family)
+                        R.id.classifyMyself -> buttonClassify.text =
+                            getString(R.string.title_classify_myself)
+                    }
+                    true
+                }
+                show()
+            }
+        }
+    }
+
+    @SuppressLint("SetTextI18n")
+    private fun showEvaluateMenu() {
+        var a: String? = ""
+        var b: String? = ""
```

Comment:
a với b là gì thế?

Reply:
2 biến để lưu 2 giá trị Evaluate. Rush or not rush với important or not important

Reply:
ý anh Đạt là đặt tên lại cho có ý nghĩa